presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -638,14 +638,14 @@ public void testGroupByNullIf()
     public void testGroupByExtract()
     {
         // whole expression in group by
-        assertQuery("SELECT EXTRACT(YEAR FROM now()), count(*) FROM orders GROUP BY EXTRACT(YEAR FROM now())");
+        assertQuery("SELECT EXTRACT(YEAR FROM orderdate), count(*) FROM orders GROUP BY EXTRACT(YEAR FROM orderdate)");
```

Comment:
This should not only make the test pass, but also make it more like "group by EXTRACT(...)".
Previously, `EXTRACT(YEAR FROM now())` should be constant-folded.

Reply:
Because of this change, i'd change the commit title to reflect that.